.circleci/config.yml

```diff
@@ -71,7 +79,7 @@ jobs:
           cache-prefix: << pipeline.parameters.build-cache-prefix >>
 
       - store_artifacts:
-          path: code/<< pipeline.parameters.build-path >>/build/reports  
+          path: code/<< pipeline.parameters.build-path >>/reports  
```

Comment:
The `build-path` is `code/analytics/build`, so looks like you need to remove "code/" as well.